samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -787,4 +787,23 @@ function toggleCategories(checkbox) {
             categoryDiv.fadeOut();
         }
     }
-}
\ No newline at end of file
+}
+
+function expandAccordion(iframId){
+    $('div#jqueryui-accordion > .ui-accordion-content').show();
+    mySetMainFrameHeight(iframId);
+    $("#collapseLink").show();
+    $("#expandLink").hide();
+    $("div#jqueryui-accordion > h3.ui-accordion-header > span").removeClass("ui-icon-triangle-1-e").addClass("ui-icon-triangle-1-s");
```

Comment:
Sorry, I haven't investigated alternative ways of accomplishing this. The worry about these lines was how fragile they could be. We could file a separate Jira issue to fix them better in the future.